tests/Feature/Services/Backup/RestoreTaskTest.php

```
@@ -78,31 +57,38 @@ function createRestoreDatabaseServer(array $attributes): DatabaseServer
-        ->shouldReceive('prepareDatabase')
-        ->once()
-        ->with(Mockery::type(\App\Services\Backup\Databases\DatabaseInterface::class), $restore->schema_name, Mockery::any())
-        ->andReturnNull();
-
-    // Mock download - create a compressed file that will be decompressed
+    // Mock download
     test()->filesystemProvider
         ->shouldReceive('download')
         ->once()
         ->with($restore->snapshot, Mockery::any())
         ->andReturnUsing(function ($snap, $destination) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$snap'. (undefined)

(UnusedFormalParameter)